### DISCUSSION TITLE
Hotkey for media control

### DISCUSSION THREAD

**Discussion by MohaElder** [ciderapp/Cider#1520] [Feature Request]
I was wondering if Cider could support hotkey for controlling media.

e.g: 'ctrl + alt + left arrow key' can go back to the previous music, 'ctrl + alt + p' can pause the music.

Thanks in advance!

**Comment by NathanielH-snek** [comment]
I agree. I have a macro on my mouse for going forward and back in audio and for some reason cider doesn't seem to support it yet?

**Comment by quacksire** [comment]
Media keys should work, how about mapping your buttons to that

**Reply by NathanielH-snek** [reply]
Play/Pause seems to work fine, but next track and previous track don't for me. Anything I should try?

**Reply by Zahin-Mohammad-plug** [reply]
Same issue here

**Reply by NathanielH-snek** [reply]
Turns out it's a bug. They work when you restart the apps after opening it for the first time
